src/main/java/org/bigml/binding/LocalCluster.java

```diff
@@ -31,82 +31,111 @@
  *
  * import org.bigml.binding.LocalCluster;
  *
- * JSONObject clusterData = BigMLClient.getInstance().
+ * JSONObject cluster = BigMLClient.getInstance().
  *                                getCluster("cluster/5026965515526876630001b2");
- * LocalCluster cluster = LocalCluster(clusterData)
- *
- * JSONObject predictors = JSONValue.parse("{\"petal length\": 3,
- *                                \"petal width\": 1,
- *                                \"sepal length\": 1,
- *                                \"sepal width\": 0.5}");
- *
- * cluster.predict(predictors)
+ * LocalCluster cluster = LocalCluster(cluster)
```

Comment:
The original code was more readable. I am not sure that local variable shadowing is a good Java practice...

Reply:
I don't understand.

Reply:
The original code used `clusterData` for the JSON object and cluster for the local cluster object -- two objects, two variables with different names. The new code use `cluster` for the JSON object, then shadows that variable by defining a new `cluster` variable to refer to the local cluster object. I think the old code was more readable and clearer as to the intentions behind it. If you think you want to stress the transient nature of the first object, just pass it into the second's constructor.

Reply:
ah, ok, I change it